### ISSUE TITLE
Trying to load pre downloaded Imagenet2012 validation set only.

### ISSUE THREAD

**Issue by LukasDauterman** [tensorflow/datasets#3154]
**What I need help with / What I was wondering**
I am trying to load only validation set for ImageNet2012, and I have already it downloaded and have been able to load it no problem in Pytorch, but I need to load it for TensorFlow models. How do I only load and use the validation set?

**What I've tried so far**
I have tried tfds builder and load but there doesn't seem to be an option for only loading the validation data set without it trying to download the training data, which I don't have the hard drive space for.

**It would be nice if...**
I could find documentation or examples of how to load pre downloaded ImageNet 2012 validation set only.

**Environment information**
(if applicable)
* Operating System: Ubuntu 18.04 running in WSL 2
* Python version: 3.7 or 3.8 not sure exactly
* `tensorflow-datasets`/`tfds-nightly` version:  tensorflow-datasets 4.2.0
* `tensorflow`/`tensorflow-gpu`/`tf-nightly`/`tf-nightly-gpu` version: tf-nightly 2.6.0-dev20210408

**Comment by Conchylicultor** [comment]
Imagenet require you to put the manually downloaded archive file in the manual dir (`~/tensorflow_datasets/download/manual/`). If you only put the validation set in the manual dir, then the TFDS dataset will be generated with only the validation set
